src/Forum/utils.ts

```diff
@@ -0,0 +1,70 @@
+import { env } from 'decentraland-commons'
+import { shorten } from '../utils/address'
+import { CollectionAttributes } from '../Collection'
+import { FullItem } from '../Item'
+import { ForumPost } from './Forum.types'
+
+// These methods are suspiciously similar to https://github.com/decentraland/builder/blob/master/src/modules/forum/utils.ts
+// It should be deleted from there and used here once we tackle https://github.com/decentraland/builder/issues/1754
+// Keep in mind the TODO above getItemEditorUrl
+
+const BUILDER_URL = env.get('BUILDER_URL', '')
+const BUILDER_SERVER_URL = env.get('BUILDER_SERVER_URL', '')
+
+export function buildCollectionForumPost(
+  collection: CollectionAttributes,
+  items: FullItem[]
+): ForumPost {
+  // We only post in English
+  return {
+    title: `Collection '${collection.name}' created by ${shorten(
+      collection.eth_address
+    )} is ready for review!`,
+    raw: `# ${collection.name}
+
+	[View entire collection](${getItemEditorUrl({ collectionId: collection.id })})
+
+	## Wearables
+
+	${items.map(toRawItem).join('\n\n')}`,
+  }
+}
+
+function toRawItem(item: FullItem) {
+  const sections = []
+  if (item.description) {
+    sections.push(`- Description: ${item.description}`)
+  }
+  if (item.rarity) {
+    sections.push(`- Rarity: ${item.rarity}`)
+  }
```

Comment:
because I'm receiving a FullItem, wouldn't this line work for both cases? I mean if I supply a TP item, rarity will be `undefined` and thus not added, but a Standard item'd work if we use it in the future

Reply:
You're actually right.